notebooks/02-ingestion/02-batch-ingestion.ipynb

```diff
@@ -262,37 +240,54 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "druid.display.sql(\"\"\"\n",
+    "sql = f'''\n",
     "SELECT channel, count(*) num_events\n",
-    "FROM \"example-wikipedia-3-batch\" \n",
-    "WHERE __time BETWEEN '2016-06-27' AND '2016-06-28'\n",
+    "FROM \"{table_name}\" \n",
+    "WHERE TIME_IN_INTERVAL(\"__time\", '2016-06-27/2016-06-28')\n",
     "GROUP BY 1 \n",
     "ORDER BY 2 DESC \n",
     "LIMIT 10\n",
-    "\"\"\")"
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1728e627-a849-4753-9bde-382f557c2584",
+   "metadata": {},
+   "source": [
+    "Run the following cell to drop the table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc6b287c-be0e-48fa-ae39-e111ae9dd965",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "druid.datasources.drop(table_name, True)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "751316a2-bd0c-43eb-a74f-fed15782f54d",
    "metadata": {},
    "source": [
-    "## Context parameters\n",
-    "Certain aspects of the ingestion can be controlled using [context parameter](https://druid.apache.org/docs/latest/multi-stage-query/reference.html#context-parameters)s. This section discussed two of the commonly used parameters:\n",
-    "\n",
-    "##### maxNumTasks\n",
-    "The Multi-stage Query Framework uses parallel workers to execute each stage of the ingestion process. Each stage creates output partitions that organize the data in preparation for the next stage. \n",
+    "Use [context parameters](https://druid.apache.org/docs/latest/multi-stage-query/reference.html#context-parameters) to control how the ingestion executes.\n",
     "\n",
-    "The input stage parallelism is limited by the input sources, as each file is processed by one of the workers. While multiple input files are split evenly among parallel worker tasks. As such a single large file cannot be parallelized at this stage. You can split very large files into smaller ones to improve parallelism for the input stage.\n",
+    "When ingesting from multiple sources at once, Druid parallelizes the task up to the number of `numTasks`.\n",
```

Comment:
parallelizes always feels like a weird verb to me, I think that "Druid creates up to the value of `numTasks` multiple tasks" or words to that effect is a little easier to read/conceptualise but I appreciate that might just be personal preference